SPECS/azurelinux-sysinfo/collect-sysinfo

```diff
@@ -220,10 +220,36 @@ def get_selinux_mode():
         ["getenforce"], capture_output=True, text=True, check=True
     ).stdout.strip()
 
+def get_system_manufacturer():
+    return subprocess.run(
+        ["cat", "/sys/devices/virtual/dmi/id/sys_vendor"], capture_output=True, text=True
```

Comment:
no `check=True` here

Reply:
I'm not sure about this as a mechanism for detecting virtual vs physical. This will show the vendor and doesn't necessarily have to be QEMU.
For example my linux hyper-v VM shows "Microsoft" under that path,

Reply:
that means your hyper-v vm is on microsoft device? if you want to capture whether the machine is a VM that is different. this simply identifies whether the **host** machine is virtual or physical. eg if you create a vm on your hyperv vm it will show QEMU under the path

Reply:
and yes i know theres no check=True there - dont think we need the service to fail in this case right now